apps/event-service/app.py

```diff
@@ -25,7 +26,25 @@ def get_events():
     res = supabase.table("events").select(
         "name,description,start,end,id").execute()
     return res.data
-
+  
+  
+@app.get("/events/<day>")
+def get_events_by_day(day: int):
+  day_begin, day_end = None, None
+  
+  if day == "1":
+    day_begin = datetime(2023,2,3,0,0,0,0)
+    day_end = datetime(2023,2,3,23,59,59,999999)
+  elif day == "2":
+    day_begin = datetime(2023,2,4,0,0,0,0)
+    day_end = datetime(2023,2,4,23,59,59,999999)
+  elif day == "3":
+    day_begin = datetime(2023,2,5,0,0,0,0)
+    day_end = datetime(2023,2,5,23,59,59,999999)
```

Comment:
um no this is not acceptable; use [timedelta](https://docs.python.org/3/library/datetime.html#timedelta-objects) or something